lib/node.js

```diff
@@ -988,24 +993,41 @@ declare class stream$Duplex extends stream$Readable {
   constructor(options?: duplexStreamOptions): void;
   cork(): void;
   end(
-    chunk?: string | Buffer,
-    encodingOrCallback?: string | (data: any) => void,
-    callback?: (data: any) => void
+    chunk?: Buffer | string,
+    encodingOrCallback?: string | Function,
+    callback?: Function
```

Comment:
The callback change here is because the callback is just a listener for the `finish` event, which receives nothing:

1. https://nodejs.org/dist/latest-v5.x/docs/api/stream.html#stream_writable_end_chunk_encoding_callback
2. https://nodejs.org/dist/latest-v5.x/docs/api/stream.html#stream_event_finish